schema/deploy/todo_data.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy todo:todo_data to pg
+-- requires: todo_app
+
+BEGIN;
+
+-- XXX Add DDLs here.
+create table if not exists todo_data (
```

Comment:
the table needs to be created on the `todo_app` schema
```suggestion
create table if not exists todo_app.todo_data (
```

Reply:
Good catch!